Lambdas/GHOAuth/OAuthLambda.swift

```diff
@@ -33,42 +33,39 @@ struct GHOAuthHandler: LambdaHandler {
         self.jsonEncoder = JSONEncoder()
     }
 
-    func handle(_ event: APIGatewayV2Request, context: LambdaContext) async throws -> APIGatewayV2Response {
+    func handle(_ event: APIGatewayV2Request, context: LambdaContext) async -> APIGatewayV2Response {
+        logger.trace("Received event: \(event)")
+
+        guard let code = event.queryStringParameters?["code"] else {
+            return .init(statusCode: .badRequest, body: "Missing code query parameter")
+        }
+
+        let accessToken: String
+
         do {
-            return try await handleThrowing(event, context: context)
+            accessToken = try await getAccessToken(code: code)
         } catch {
-            do {
-                try await discordClient.createMessage(
-                    channelId: OAuthLambdaConstants.Channels.logs.id,
-                    payload: .init(embeds: [.init(
-                        title: "GHOAuth lambda top-level error",
-                        description: String("\(error)".prefix(4_000)),
-                        color: .red
-                    )])
-                ).guardSuccess()
-            } catch {
-                logger.error("DiscordClient logging error", metadata: [
-                    "error": "\(error)"
-                ])
-            }
-            throw error
+            return .init(statusCode: .badRequest, body: "Error getting access token: \(error)")
```

Comment:
Much better, but log the error using the logger, and just return a generic error message to users.
It'll both make sure we don't leak anything, and also a user can't do much with the error anyway/
```swift
logger.error("Error while getting the code", metadata: ["error": "\(error)"])
return .init(statusCode: .badRequest, body: "Error getting access token")
```